### PR TITLE
feat: add trpc

### PR DESCRIPTION
**Pull request by mrevanzak**

### CODE REVIEW

File: tRPC/prisma/schema.prisma [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+generator client {
+  provider = "prisma-client-js"
+}
+
+datasource db {
+  provider = "postgresql"
+  // NOTE: When using postgresql, mysql or sqlserver, uncomment the @db.Text annotations in model Account below
+  // Further reading:
+  // https://next-auth.js.org/adapters/prisma#create-the-prisma-schema
+  // https://www.prisma.io/docs/reference/api-reference/prisma-schema-reference#string
```
Comment (theodorusclarence) [inline tRPC/prisma/schema.prisma:13]: based on this comment, should we uncomment `// @db.Text` schema for these?

image

File: tRPC/src/env.mjs
Comment (theodorusclarence) [inline tRPC/src/env.mjs:1]: interesting approach, is this code taken from t3 or trpc docs?

if it is from t3, please add reference using JSDoc on top of the file (line 1)

```js
/** @see https://link.here */
```

**Review by theodorusclarence — CHANGES_REQUESTED** [review]

File: tRPC/prisma/schema.prisma (hunk 1 above)
Comment (mrevanzak) [reply tRPC/prisma/schema.prisma:13]: yes my bad

File: tRPC/src/env.mjs
Comment (mrevanzak) [reply tRPC/src/env.mjs:1]: yes straight from t3

**Comment by theodorusclarence** [comment]
looks good, I’ll try to test them soon

**Comment by theodorusclarence** [comment]
tried it, looks good!  Thank you @mrevanzak

File: tRPC/trigger.sh [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+  "src/server/auth.ts"
+  "src/server/db.ts"
+  "src/utils/api.ts"
+  "src/env.mjs"
+)
+for i in "${files[@]}"
+do
+  encoded_file=$(echo "$i" | sed -e 's/\[/%5B/g' -e 's/\]/%5D/g')  # URL-encode the file path
+  echo "Downloading... $i"
+  curl -LJs -o $i https://raw.githubusercontent.com/mrevanzak/expansion-pack/main/$DIRNAME/$encoded_file
```
Comment (theodorusclarence) [inline tRPC/trigger.sh]: last one is only the repository link, after you update them, I'll merge this PR. Thanks

Comment (mrevanzak) [reply tRPC/trigger.sh]: done sir @theodorusclarence

**Pull request merged**